tesseract_motion_planners/descartes/include/tesseract_motion_planners/descartes/impl/descartes_collision_edge_evaluator.hpp

```diff
@@ -54,14 +54,16 @@ DescartesCollisionEdgeEvaluator<FloatType>::DescartesCollisionEdgeEvaluator(
   , debug_(debug)
 {
   discrete_contact_manager_->setActiveCollisionObjects(active_link_names_);
-  discrete_contact_manager_->setCollisionMarginData(collision_check_config_.collision_margin_data,
-                                                    collision_check_config_.collision_margin_override_type);
+  discrete_contact_manager_->setCollisionMarginData(
+      collision_check_config_.contact_manager_config.margin_data,
+      collision_check_config_.contact_manager_config.margin_data_override_type);
   discrete_contact_manager_->setIsContactAllowedFn(
       [this](const std::string& a, const std::string& b) { return isContactAllowed(a, b); });
 
   continuous_contact_manager_->setActiveCollisionObjects(active_link_names_);
-  continuous_contact_manager_->setCollisionMarginData(collision_check_config_.collision_margin_data,
-                                                      collision_check_config_.collision_margin_override_type);
+  continuous_contact_manager_->setCollisionMarginData(
+      collision_check_config_.contact_manager_config.margin_data,
+      collision_check_config_.contact_manager_config.margin_data_override_type);
```

Comment:
Note: Need to look at this further to because it is not applying the config due to the local isContactAllowed. May need slight re-configuring.

Reply:
The local acm and method is removed and now leverages the config.